src/components/SignUpForm/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import Link from 'next/link'
+
+export default function SignUpForm() {
+  return (
+    <section className="sign-up-form">
+      <header className="sign-up-form__header">Register</header>
+      <div className="sign-up-form__body">
+        <form action="" className="form">
```

Comment:
Since we can reuse this `<form/>`again in another component like <SignInForm/>. Can we make a reusable component instead?